lib/public/views/Runs/RunPerDataPass/RunsPerDataPassOverviewPage.js

```diff
@@ -46,14 +49,19 @@ export const RunsPerDataPassOverviewPage = ({ runs: { perDataPassOverviewModel }
         }), { profiles: 'runsPerDataPass' }),
     };
 
+    const commonTitle = h('h2', { style: 'white-space: nowrap;' }, 'Physics Runs');
+
     return h('', [
-        h('.flex-row.justify-between.items-center', [
-            dataPass.match({
-                Success: (payload) => h('h2', `Good, physics runs of ${payload.name}`),
-                Failure: () => h('h2.danger', 'Failed to fetch Data Pass information'),
-                Loading: () => h('.p1', spinner({ size: 2, absolute: false })),
-                NotAsked: () => h('h2', 'Good, physics runs'),
-            }),
+        h('.flex-row.justify-between.items-center.g2', [
+            h(
+                '.flex-row.g1.items-center',
+                dataPass.match({
+                    Success: (payload) => breadcrumbs(commonTitle, h('h2', payload.name)),
+                    Failure: (payload) => [commonTitle, tooltip(h('.f3', iconWarning()), errorAlert(payload))],
+                    Loading: () => [commonTitle, spinner({ size: 2, absolute: false })],
+                    NotAsked: () => [commonTitle, tooltip(h('.f3', iconWarning()), errorAlert([{ detail: 'No data was asked for' }]))],
```

Comment:
2 things:
- I would not use errorAlert in tooltip, sorry if I was unclear about it, I think that a simple text is enough
- The message brings no info at all, it was better before, like `Failed to fetch data pass name` for example...

Reply:
For not asked change to simple text, for Failure I'd keep `errorAlert` as it's only handy method to display actual error